CONTRIBUTING.md

```diff
@@ -46,76 +46,70 @@ Hi! We're really excited that you are interested in contributing to ACPaaS UI. B
-# install nested package.json dependencies
-$ npm run bootstrap
-# build into dist folder
-$ npm run build
-```
-
-Additionally, to run the examples locally:
-
-```bash
-# update the styleguide from the individual components
-$ npm run examples
-# run the styleguide examples application
-$ npm run styleguide
+```shell
+npm install
+npm run build:all
+npm start
 ```
 
 ## Using Docker
 
 Additionally to running this project locally, the repository also offers you the option to develop using Docker containers.
 To start your service, execute:
 
-```sh
+```shell
 docker-compose build
 docker-compose up -d
 ```
 
 When the service is up, use following commands to execute bash inside your container:
 
-```sh
+```shell
 docker ps
 docker exec -it ${YOUR_CONTAINER_ID} sh
 ```
 
 From there you can run all necessary npm commands for development.
 
-## Project Structure
+## Creating New Packages
 
-- `dist`: contains the build output for the components
-- `examples`: contains the build output for the examples
-- `packages`: the component sources
-    - `some-package/`: sources for a component package (one or more components)
-        - `lib/src/some-component/`: sources for a component
-        - `examples/`: examples (documentation) as shown in the styleguide app
-        - `package.json`: dependencies needed by this package
-        - `README.md`: documentation for this package, linked from the main [README.md](README.md)
-- `scripts`: helper shell scripts used by the npm tasks
-- `styleguide`: sources for the examples web app (but not the actual examples)
+Packages are essentially Angular libraries that follow Angular standard.
+More info on how to create Angular libraries can be found [here](https://angular.io/guide/libraries).
 
-[github-issues]: https://github.com/digipolisantwerp/acpaas-ui_angular/issues
+To create a new library as part of the ACPAAS UI, run:
 
-## Creating New Packages
+````shell
+ng generate library aui-<package>
+````
+
+To build and test your package, respectively run:
+````shell
+ng build aui-<package>
+ng test aui-<package>
+````
+
+Make sure to edit the `package.json` file of the library to set correct library name and publishing settings.
+
+Once your library is created, you may import the library module in the `packages/styleguide/src/aui/aui.module.ts`.
+After this, add a demo page component to the examples module: `packages/styleguide/src/examples/pages`
 
-Use `npm run package some-other-package` to create a new package folder.
-After making changes to your package, you should run `example=some-other-package npm run build:package` to see your changes.
+## Project structure
+- `packages/aui-<package>`: Acpaas UI package/library
+- `packages/styleguide`: Angular application featuring demo's of the packages
+    - `src/aui/aui.module.ts`: Shared module that imports/exports all needed ACPAAS packages for demo purposes
+    - `src/examples`: Example module containing pages and routes for the demos.
 
 ## Releases
```

Comment:
There should be another section under releases called "Publishing" with step by step instructions for how to publish a new version, with a remark there that only the ACPaaS UI team publishes new versions.

Reply:
For now it will still be a bit of tinkering as to how a publish works and which steps we need to execute, thus I was mark this as resolved, although we will complete this section once we've figured out how to publish exactly.